### DISCUSSION TITLE
Collect additional metrics for harbor

### DISCUSSION THREAD

**Discussion by chlins** [goharbor/harbor#17806] [Ideas]
Harbor has supported expose the metrics of harbor, currently existed metrics can be found [here](https://goharbor.io/docs/main/administration/metrics/), but we also got many requirement issues from the community to ask for adding other metrics, so let's discuss in this place for new metrics, feel free to leave your comments.

**Comment by yuangeqingian** [comment]
1: Replication task details such as registry name, trigger type, execution status, execution total count
2: Replication total count,
3: Replication execution total count, succeed/failure/stop count

**Comment by ruby-rhino** [comment]
Question: Has these Replication metrics been added at all to Harbor Version v2.3.3-a0a9ed8a and above?
